vllm/distributed/kv_events.py

```diff
@@ -44,16 +45,16 @@ class KVCacheEvent(
 
 
 class BlockStored(KVCacheEvent):
-    block_hashes: list[int]
-    parent_block_hash: Optional[int]
+    block_hashes: list[Union[BlockHash, int]]
```

Comment:
It might be nice to define a new type for these `ExternalBlockHash = Union[bytes, int]`?